src/Components/Elasticsearch/EsProductDefinition.php

```diff
@@ -239,7 +238,7 @@ private function buildCoalesce(array $fields, Context $context): string
             }
         }
 
-        return substr($coalesce, 0, -1) . ')';
+        return mb_substr($coalesce, 0, -1) . ')';
```

Comment:
why this change?

Reply:
we should disable that 😂

Reply:
As far as I saw you used it in other classes as well, keep it similar I would say. Only diff is apperently that it will handle UTF-8 encoding better. You can disable it in the php-cs-fixer file tho, i am emotionless in that topic :laughing: